pkg/proc/moduledata.go

```diff
@@ -1,13 +1,13 @@
 package proc
 
-// delve counterpart to runtime.moduledata
-type moduleData struct {
+// ModuleData counterpart to runtime.moduleData
+type ModuleData struct {
```

Comment:
Why export this when all members are unexported without accessor methods?

Reply:
It's for storing it somewhere else and then passing it to RuntimeTypeToDIE.